src/control/nav2_dwa_controller/CMakeLists.txt

```diff
@@ -24,7 +28,6 @@ include_directories(
 set(executable_name dwa_controller)
 
 add_executable(${executable_name}
-  src/dwa_controller.cpp
```

Comment:
If the intention is to implement the DwaController using the source code under nav2_dwb_controller, then that source should be under nav2_dwa_controller.

Reply:
That was indeed my intent.

Reply:
I removed the changes to the DwaController. I'll leave nav2_dwb_controller as a peer until it is clear whether it is a component of DwaController or a standalone controller.